tests/acceptance/general/UpgradePathsCest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Tests edge cases when upgrading between specific ConvertKit Plugin versions.
+ *
+ * @since   1.5.0
+ */
+class UpgradePathsCest
+{
+	/**
+	 * Check that ConvertKit Settings stored on a WPForms Form using < 1.5.0 are correctly
+	 * migrated to the WPForms > Settings > Integrations tab.
+	 *
+	 * @since   1.5.0
+	 *
+	 * @param   AcceptanceTester $I  Tester.
+	 */
+	public function testMigrateFormSettingsToIntegration(AcceptanceTester $I)
+	{
+		// Activate WPForms.
+		$I->activateThirdPartyPlugin($I, 'wpforms-lite');
+
+		// Create Form, as if it were created with this Plugin < 1.5.0.
+		$wpFormsID = $I->havePostInDatabase(
+			[
+				'post_type'    => 'wpforms',
+				'post_status'  => 'publish',
+				'post_title'   => 'Migrate form',
+				'post_name'    => 'migrate-form',
+				'post_content' => '{"fields":[{"id":"0","type":"name","label":"Name","format":"first-last","description":"","required":"1","size":"medium","simple_placeholder":"","simple_default":"","first_placeholder":"","first_default":"","middle_placeholder":"","middle_default":"","last_placeholder":"","last_default":"","css":""},{"id":"1","type":"email","label":"Email","description":"","required":"1","size":"medium","placeholder":"","confirmation_placeholder":"","default_value":false,"filter_type":"","allowlist":"","denylist":"","css":""},{"id":"2","type":"textarea","label":"Comment or Message","description":"","size":"medium","placeholder":"","limit_count":"1","limit_mode":"characters","default_value":"","css":""},{"id":"3","type":"text","label":"Tag ID","description":"","size":"medium","placeholder":"","limit_count":"1","limit_mode":"characters","default_value":"","input_mask":"","css":""}],"id":"2","field_id":4,"settings":{"be_convertkit_api":"' . $_ENV['CONVERTKIT_API_KEY'] . '","be_convertkit_form_id":"' . $_ENV['CONVERTKIT_API_FORM_ID'] . '","be_convertkit_field_first_name":"0","be_convertkit_field_email":"1","form_title":"Simple Contact Form","form_desc":"","submit_text":"Submit","submit_text_processing":"Sending...","form_class":"","submit_class":"","ajax_submit":"1","notification_enable":"1","notifications":{"1":{"email":"{admin_email}","subject":"New Entry: Simple Contact Form","sender_name":"convertkit","sender_address":"{admin_email}","replyto":"{field_id=\"1\"}","message":"{all_fields}"}},"confirmations":{"1":{"type":"message","message":"<p>Thanks for contacting us! We will be in touch with you shortly.<\/p>","message_scroll":"1","redirect":""}},"antispam":"1","form_tags":[]},"meta":{"template":"simple-contact-form-template"}}',
```

Comment:
It might be worth it to split this out into another file so we can have it nicely formatted.